assets/blueprints/kv_store/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+use scrypto::prelude::*;
+
+#[derive(ScryptoSbor)]
+struct TestType {
+    b: u32,
+}
+
+#[blueprint]
+#[experimental_types(TestType)]
```

Comment:
I take it that this isn't the final name of the attribute and we want to get the scenario out first and then change the attribute name since it doesn't impact anything on ledger?

Reply:
That's right!